common/lumens.js

```diff
@@ -220,3 +220,18 @@ export function circulatingSupply() {
     return new BigNumber(totalLumens).minus(noncirculatingSupply);
   });
 }
+
+export function totalSupplySum() {
+  return Promise.all([
+    getUpgradeReserve(),
+    feePool(),
+    sdfAccounts(),
+    circulatingSupply(),
+  ]).then((balances) => {
+    return reduce(
+      balances,
+      (sum, balance) => sum.add(balance),
```

Comment:
❓ - do we also need to convert the computation of each of the promise functions (getUpgradeReserve, feePool, etc) to BigNumber so we don't lose precision? Probably not, but thought worth checking.

Reply:
good point, since not all the values from the promise are returned as bignumber I changed just to be safe
https://github.com/stellar/dashboard/pull/182/commits/06e4a72d327ccb7091b82a464212cc4a9a18ffce